### PR TITLE
Fix UTC format for logs

### PR DESCRIPTION
**Pull request by xnull**
## Overview

Description:
Fix mistake in log format

Why should this be merged: 
Fix mistake in log format. Improved log format

Related issue(s) (if applicable): #<number>

## Checklist (Definition of Done):

- [x] There are no TODOs left in the code
- [x] [Coding conventions](https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines) (e.g. for logging, unit tests) have been followed
- [x] Change is covered by automated tests
- [x] Public API has Javadoc

### CODE REVIEW

File: test/src/test/resources/logback-test.xml [hunk 1]
```diff
@@ -46,7 +46,7 @@
 
     <root level="INFO">
         <!--<appender-ref ref="FILE" />-->
-        <!--<appender-ref ref="STDOUT" />-->
+        <appender-ref ref="STDOUT" />
```
Comment (zalokhan) [inline test/src/test/resources/logback-test.xml]: Keep this commented. Else, it clutters Travis.

**Comment by xnull** [comment]
My proposal for logs format:
 - server: `${date} | ${address:port} 15 chars | ${logLevel} | ${threadName} 15 chars | ${className} 30 chars |  ${message}`

 - tests: `${date} | ${address:port} 15 chars | ${logLevel} | ${threadName} 15 chars | ${className} 30 chars | **${methodName:line} 20 chars** |  ${message}`

**Comment by xnull** [comment]
This is how it looks in tests:
```
2018-11-15T22:37:22.023Z | INFO  |       test-main |  org.corfudb.util.MetricsUtils | csReportingSetup:153 | Corfu metrics collection and all reporting types are disabled
2018-11-15T22:37:22.024Z | INFO  |       test-main |            o.c.i.ServerContext |   generateNodeId:249 | No Node Id, setting to new Id=6mslwwFnTT2_ZlzPJR_NRg
2018-11-15T22:37:22.097Z | INFO  |       test-main |            o.c.i.ServerContext | SingleNodeLayout:318 | getNewSingleNodeLayout: Bootstrapping with cluster Id 203146b3-e822-4cc1-9e24-5cc69759c792 [IDFGs-giTMGeJFzGl1nHkg]
2018-11-15T22:37:22.166Z | WARN  |       test-main |            o.c.i.LogUnitServer |           <init>:107 | Log unit opened in-memory mode (Maximum size=1.9 GB). This should be run for testing purposes only. If you exceed the maximum size of the unit, old entries will be AUTOMATICALLY trimmed. The unit WILL LOSE ALL DATA if it exits.
2018-11-15T22:37:22.190Z | INFO  |       test-main |            o.c.i.ServerContext | ManagementLayout:475 | saveManagementLayout: Updated to new layout at epoch 0
2018-11-15T22:37:22.190Z | WARN  |       test-main |            o.c.i.ServerContext | ManagementLayout:467 | saveManagementLayout: Attempted to update with null layout
2018-11-15T22:37:22.195Z | INFO  | tectionWorker-0 |          o.c.i.ManagementAgent |    lambda$null$1:255 | triggerSequencerBootstrap: a bootstrap task is triggered.
2018-11-15T22:37:22.196Z | INFO  | tectionWorker-0 |  org.corfudb.util.MetricsUtils | csReportingSetup:153 | Corfu metrics collection and all reporting types are disabled
2018-11-15T22:37:22.196Z | INFO  | tectionWorker-0 | o.corfudb.runtime.CorfuRuntime |           <init>:572 | Corfu runtime version source(c7c9a2c) initialized.
2018-11-15T22:37:22.199Z | INFO  | tectionWorker-0 | o.corfudb.runtime.CorfuRuntime |          connect:911 | Connecting to Corfu server instance, layout servers=[test:0]
2018-11-15T22:37:22.199Z | INFO  |       test-main | o.corfudb.runtime.CorfuRuntime |          connect:911 | Connecting to Corfu server instance, layout servers=[test:0]
2018-11-15T22:37:22.367Z | INFO  | onPool-worker-2 | o.corfudb.runtime.CorfuRuntime |   checkClusterId:760 | Connected to new cluster IDFGs-giTMGeJFzGl1nHkg
2018-11-15T22:37:22.371Z | INFO  | onPool-worker-1 | o.corfudb.runtime.CorfuRuntime |   checkClusterId:760 | Connected to new cluster IDFGs-giTMGeJFzGl1nHkg
```

**Comment by xnull** [comment]
Server:
```
2018-11-15T22:37:22.024Z | localhost:9000 | INFO  |       test-main |            o.c.i.ServerContext  | No Node Id, setting to new Id=6mslwwFnTT2_ZlzPJR_NRg
```

**Comment by apavan** [comment]
I would add more buffer as below... 

server: ${date} | ${address:port} 30 chars | ${logLevel} | ${threadName} 30 chars | ${className} 30 chars | ${message}

tests: ${date} | ${logLevel} | ${threadName} 30 chars | ${className} 30 chars | ${methodName:line} 30 chars | ${message}

and also make sure there is no limit on the line.. we should assume it should handle atleast 10000-99000

**Comment by Maithem** [comment]
Is there really no dynamic way of figuring this out?

**Comment by xnull** [comment]
There is no limit for the line length

**Comment by xnull** [comment]
I just updated log format. 
 - server:
  `date | level | serverName + threadName 50 letters`| loggerName 30 letters | message`
```
2018-11-16T20:25:20.796Z | DEBUG |   xtwnny-corfu-node52814:52814 | ManagementService |            o.c.i.ServerContext | saveManagementLayout: Ignoring layout because new epoch 0 <= old epoch 0
2018-11-16T20:25:20.798Z | DEBUG |   xtwnny-corfu-node52814:52814 | DetectionWorker-1 |        o.c.i.m.HealingDetector | Unresponsive members to poll, []
2018-11-16T20:25:20.799Z | DEBUG |   xtwnny-corfu-node52814:52814 | DetectionWorker-2 |        o.c.i.m.FailureDetector | Responsive members to poll, [xtwnny-corfu-node52816:52816, xtwnny-corfu-node52814:52814, xtwnny-corfu-node52815:52815]
```
 - tests:
 `date | level | thread 15 letters | className+methodName+line 50 letters | message 
```
2018-11-16T20:50:51.617Z | INFO  | :0-DetectionWorker-0 | b.infrastructure.ManagementAgent.lambda$null$1:255 | triggerSequencerBootstrap: a bootstrap task is triggered.
2018-11-16T20:50:51.618Z | INFO  | :0-DetectionWorker-0 | orfudb.util.MetricsUtils.metricsReportingSetup:153 | Corfu metrics collection and all reporting types are disabled
2018-11-16T20:50:51.618Z | INFO  | :0-DetectionWorker-0 |        org.corfudb.runtime.CorfuRuntime.<init>:572 | Corfu runtime version source(f7ccc81) initialized.
2018-11-16T20:50:51.620Z | INFO  | :0-DetectionWorker-0 |       org.corfudb.runtime.CorfuRuntime.connect:911 | Connecting to Corfu server instance, layout servers=[test:0]
```

**Pull request merged**